pypresence/client.py

```diff
@@ -18,37 +18,51 @@ def __init__(self, *args, **kwargs):
     def register_event(self, event: str, func: callable, args=None):
         if args is None:
             args = {}
+
         if inspect.iscoroutinefunction(func):
             raise NotImplementedError
+
         elif len(inspect.signature(func).parameters) != 1:
             raise ArgumentError
+
         self.subscribe(event, args)
         self._events[event.lower()] = func
 
     def unregister_event(self, event: str, args=None):
         if args is None:
             args = {}
+
         event = event.lower()
+
         if event not in self._events:
             raise EventNotFound
+
         self.unsubscribe(event, args)
         del self._events[event]
 
     # noinspection PyProtectedMember
     def on_event(self, data):
         if self.sock_reader._eof:
             raise PyPresenceException('feed_data after feed_eof')
+
         if not data:
             return
+
```

Comment:
Honestly not sure if these newlines across this feel are really necessary, seems like personal preference but oh well, doesn't bother me too much.

Reply:
I like having space between block help for readability, can be removed it you prefer tho

Reply:
It's just that for me it does little to improve readability. Entirely a preference thing, it's fine to keep it :)